gradle/lib/dependabot/gradle/file_parser/property_value_finder.rb

```diff
@@ -86,7 +86,7 @@ def property_details(property_name:, callsite_buildfile:)
           all_files = [callsite_buildfile, top_level_buildfile].concat(
             FileParser.find_includes(callsite_buildfile, dependency_files),
             FileParser.find_includes(top_level_buildfile, dependency_files)
-          )
+          ).compact()
```

Comment:
I checked, and `FileParser.find_includes()` already protects against hitting NullPointer:
https://github.com/dependabot/dependabot-core/blob/5dc907e3148961c74ddc0b4494730f8e86118f79/gradle/lib/dependabot/gradle/file_parser.rb#L51

So this should be the only code change needed 👍